shuffle-storage/src/main/java/com/alibaba/flink/shuffle/storage/partition/LocalFileMapPartitionFactory.java

```diff
@@ -56,13 +56,14 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(LocalFileMapPartitionFactory.class);
 
-    @GuardedBy("lock in data store")
+    protected final Object lock = new Object();
+
+    @GuardedBy("lock")
     protected final Queue<StorageMeta> ssdStorageMetas = new ArrayDeque<>();
 
-    @GuardedBy("lock in data store")
+    @GuardedBy("lock")
     protected final Queue<StorageMeta> hddStorageMetas = new ArrayDeque<>();
 
-    @GuardedBy("lock in data store")
     protected final UsableStorageSpaceInfo usableSpace = new UsableStorageSpaceInfo(0, 0);
```

Comment:
Why didn't we replace `@GuardedBy("lock in data store")` by `@GuardedBy("lock")` for `usableSpace`?

Reply:
The fields in UsableStorageSpaceInfo are now volatile. So there is no need to guard it by lock.